webpack/webpack.config.js

```diff
@@ -57,11 +57,13 @@ module.exports = function(env, { /* @if jasmine || tape || mocha*/runTest, /* @e
     entry: test ? './test/all-spec./* @if babel */js/* @endif *//* @if typescript */ts/* @endif */' :  './src/main./* @if babel */js/* @endif *//* @if typescript */ts/* @endif */',
     // @endif
     // @if !jasmine && !tape && !mocha
-    entry: './src/main./* @if babel */js/* @endif *//* @if typescript */ts/* @endif */',
+    entry: {
+      entry: './src/main./* @if babel */js/* @endif *//* @if typescript */ts/* @endif */'
+    },
     // @endif
     output: {
       path: path.resolve(__dirname, 'dist'),
-      filename: 'entry-bundle.js'
+      filename: production ? '[name].[contenthash].bundle.js' : '[name].bundle.js'
```

Comment:
Probably need at least make test mode to use `entry-bundle.js`, currently the browser test (using my browser-do) asssumes the bundle name as `entry-bundle.js` regardless of bundler (webpack or dumber).

Probably needs to karma to replace the simple browser-do, as karma has better watch mode and coverage report  support.

Reply:
ok in non prod it still should be entry-bundle....hmmm